README.rst

```diff
@@ -1,7 +1,108 @@
-========
-Overview
-========
+=================
+pytest-remotedata
+=================
 
-This plugin enables control over the access of remote data when running
-pytests. It has been moved from the core astropy project since it is of use
-more generally. It is currently WIP.
+This package provides a plugin for the `pytest`_ framework that allows
+developers to control unit tests that require access to data from the internet.
+It was originally part of the `astropy`_ core package, but has been moved to a
+separate package in order to be of more general use.
+
+.. _pytest: https://pytest.org/en/latest/
+.. _astropy: https://astropy.org/en/latest/
+
+
+Motivation
+----------
+
+Many software packages provide features that require access to data from the
+internet. These features need to be tested, but unit tests that access the
+internet can dominate the overall runtime of a test suite.
+
+The ``pytest-remotedata`` plugin allows developers to indicate which unit tests
+require access to the internet, and to control when and whether such tests
+should execute as part of any given run of the test suite.
+
+Installation
+------------
+
+The ``pytest-remotedata`` plugin can be installed using ``pip``::
+
+    $ pip install pytest-remotedata
+
+It can also be installed from source::
```

Comment:
``It is also possible to install the latest development version from the repository:``

(because pip is from source too)